### PR TITLE
Added pace (https://github.com/HubSpot/pace) definitions

### PR DESCRIPTION
**Pull request by borislavjivkov**
This pull request adds definition file for pace as requested in #7456

### CODE REVIEW

File: pace/pace.d.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
+// Type definitions for pace v0.7.5
+// Project: https://github.com/HubSpot/pace
+// Definitions by: Borislav Zhivkov <https://github.com/borislavjivkov>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface PaceOptions {
```
Comment (vvakame) [inline pace/pace.d.ts]: http://definitelytyped.org/guides/best-practices.html#ghost-modules
please check this document.

File: pace/pace.d.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
+// Type definitions for pace v0.7.5
+// Project: https://github.com/HubSpot/pace
```
Comment (vvakame) [inline pace/pace.d.ts]: https://www.npmjs.com/package/pace same project?

Comment (borislavjivkov) [inline pace/pace.d.ts]: They are different projects.

Comment (vvakame) [inline pace/pace.d.ts]: umm... `pace` is taked in npm.
Do you like `HubSpot-pace/HubSpot-pace.d.ts` ?

Comment (borislavjivkov) [inline pace/pace.d.ts]: Sounds good to me. I will also change the names of the modules accordingly.

**Comment by borislavjivkov** [comment]
I updated names to HubSpot-pace. Let me know if I need to make additional changes.

**Comment by vvakame** [comment]
thanks!

**Pull request merged**